contributing-guides/style-guide.md

```diff
@@ -66,16 +67,21 @@ Keep the following guidelines in mind when choosing tokens:
 - In case of a possible reference both to a file or a directory,
   use `{{path/to/file_or_directory}}`.
 
-Extensions
+### Extensions
 
 - If a particular extension is expected for the file, append it.
   For example, `unrar x {{compressed.rar}}`.
 - In case a generic extension is needed, use `{{.ext}}`, but **only** if an extension is required.
   For instance, in `find.md`'s example "Find files by extension" (`find {{root_path}} -name '{{*.ext}}'`)
   using `{{*.ext}}` explains the command without being unnecessarily specific;
   while in `wc -l {{file}}` using `{{file}}` (without extension) is sufficient.
+  
+### Flags
+
+- Always use long flags when applicable, for example instead of `fish -c`, use `fish --command`, so the reader can understand the meaning of a flag.
```

Comment:
This is a convention yes, but it doesn't always hold true. It's actually a case-by case basis. For example, on pages like `ls` or `git commit` which are extremely frequently used we do not actually recommend `--long-arguments`.